solutions/usaco-1159.mdx

```diff
@@ -144,6 +144,41 @@ public class ConnectingTwoBarns {
 ```
 
 </JavaSection>
+<PySection>
+
+```python
+def visit_neighbour(start):
+
+	if not visited[start]:
+		visited[start] = True
+
+	for neighbour in roads[start]:
+		visit_neighbour(neighbour)
+
+for _ in range(int(input())):
+
+	N, M = map(int, input().split())
+	visited = [False] * (N)
+	roads = [[] for _ in range(N)]
+
+	for i in range(M):
+
```

Comment:
what are these random spaces for?

Reply:
Will remove it

Reply:
I just wanted to make it more readable separating the loops and conditional statements.